src/components/Banner/Banner.stories.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { StoryFn } from '@storybook/react';
+import Banner, { BannerProps } from "./Banner";
```

Comment:
I cry a little bit inside seeing both default and named exports 😂 
But I guess it's ok

Reply:
embrace it ❤️

Reply:
Will try 😄